liba/src/external/openbsd/s_logbf.c

```diff
@@ -0,0 +1,42 @@
+/* s_logbf.c -- float version of s_logb.c.
```

Comment:
Hello,
We prefer to use OpenBSD 4.9's libm version (see 3adc5d9fac733fe04a4), and also not modifying the external files. Could you please replace this file with [s_logbf.c.zip](https://github.com/numworks/epsilon/files/4254640/s_logbf.c.zip)?

Reply:
Sadly, that logbf (and logb) is wrong for subnormal inputs (it returns a constant). Could you import just this file from openlibm?

I started a discussion on the OpenBSD misc mailing list regarding this bug: https://marc.info/?l=openbsd-misc&m=158272385126733&w=2